frontend/src/features/archived-items/crawl-pending-exclusions.ts

```diff
@@ -72,7 +72,7 @@ export class CrawlPendingExclusions extends LiteElement {
       <btrix-badge variant=${this.total ? "danger" : "neutral"} class="ml-1">
         ${this.total
           ? this.total > 1
-            ? msg(str`+${this.total.toLocaleString()} URLs`)
+            ? msg(str`+${this.total} URLs`)
```

Comment:
This should be rendered formatted as well, I think — lemme start a branch for updates to this

Reply:
#2160